cmd/db.go

```diff
@@ -34,7 +38,127 @@ var seedCmd = &cobra.Command{
 	Long:  "seed database",
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("seed database")
-		// Placeholder for db seed code.
+		config.InitConfig()
+		db.InitDB()
+		// Changes for seeding API data; local testing
+
+		rhAccount1 := &model.RHAccount{
```

Comment:
I think we should seed this **TEST** data from somewhere else. This should be the seeding location for the data which is actually used in production. Example - [rule.json](https://github.com/RedHatInsights/ros-backend/blob/main/seed.d/rules.json) which we use in production of ros-backend.

Reply:
As discussed, updated the seed command to `apiseedtest` for now